src/pages/journal/components/test-outcome/__tests__/test-outcome.spec.ts

```diff
@@ -25,6 +31,9 @@ describe('Test Outcome', () => {
         fixture = TestBed.createComponent(TestOutcomeComponent);
         component = fixture.componentInstance;
       });
+    store$ = TestBed.get(Store);
```

Comment:
Spying on `dispatch`, but there's no assertion about it being called. Is this just to prevent the default behaviour?

Reply:
It's to prevent failure due to it being referenced an not used.  It is required though.

Reply:
Added it to remove complaints that store wasn't used (although it was required).